_data/capability-matrix.yml

```diff
@@ -50,8 +50,8 @@ categories:
             l3: "Uses Flink's keyBy for key grouping. When grouping by window in streaming (creating the panes) the Flink runner uses the Beam code. This guarantees support for all windowing and triggering mechanisms."
           - class: spark
             l1: 'Partially'
-            l2: group by window in batch only
-            l3: "Uses Spark's groupByKey for grouping. Grouping by window is currently only supported in batch."
+            l2: support for grouping by panes (streaming) is a work in progress.
+            l3: Using groupByKey for grouping, but only if the pipeline explicitly calls for GroupByKey or the model forces it. For efficient group-compute see Combine.
```

Comment:
Say something positive about batch first, like:

"Full support for in batch mode. GroupByKey with multiple trigger firings in streaming mode is a work in progress."

And then I don't think you actually need to teach users about how to program against it here. The statement is likely true for most runners. If you just say "Using Spark's \`groupByKey\`" that tells users what they need to know, if they are familiar with Spark. (do use code font, I'd say)